editor/script_create_dialog.cpp

```diff
@@ -434,6 +434,13 @@ void ScriptCreateDialog::_path_changed(const String &p_path) {
 		return;
 	}
 
+	String path_error = ScriptServer::get_language(language_menu->get_selected())->validate_path(p);
+	if (path_error != "") {
+		_msg_path_valid(false, TTR(path_error));
```

Comment:
That won't work, `TTR()` is to define a string to translate when parsing *the source code*. I doubt that `editor/translations/extract.py` will know what to make of it :)

Just enclose the actual string in `validate_path` with `TTR()`.